client/my-sites/site-settings/publishing-tools/post-by-email.tsx

```diff
@@ -0,0 +1,168 @@
+import { Button, FormLabel } from '@automattic/components';
+import { localizeUrl } from '@automattic/i18n-utils';
+import { ToggleControl } from '@wordpress/components';
+import clsx from 'classnames';
+import { useTranslate } from 'i18n-calypso';
+import ClipboardButtonInput from 'calypso/components/clipboard-button-input';
+import FormFieldset from 'calypso/components/forms/form-fieldset';
+import SupportInfo from 'calypso/components/support-info';
+import JetpackModuleToggle from 'calypso/my-sites/site-settings/jetpack-module-toggle';
+import { useGetPostByEmail } from 'calypso/my-sites/site-settings/publishing-tools/hooks/use-get-post-by-email';
+import { useRegeneratePostByEmailMutation } from 'calypso/my-sites/site-settings/publishing-tools/hooks/use-regenerate-post-by-email-mutation';
+import { useTogglePostByEmailMutation } from 'calypso/my-sites/site-settings/publishing-tools/hooks/use-toggle-post-by-email';
+import { useDispatch, useSelector } from 'calypso/state';
+import { regeneratePostByEmail as jetpackRegeneratePostByEmail } from 'calypso/state/jetpack/settings/actions';
+import { errorNotice, successNotice } from 'calypso/state/notices/actions';
+import isJetpackModuleActive from 'calypso/state/selectors/is-jetpack-module-active';
+import isJetpackModuleUnavailableInDevelopmentMode from 'calypso/state/selectors/is-jetpack-module-unavailable-in-development-mode';
+import isJetpackSiteInDevelopmentMode from 'calypso/state/selectors/is-jetpack-site-in-development-mode';
+import isRegeneratingJetpackPostByEmail from 'calypso/state/selectors/is-regenerating-jetpack-post-by-email';
+import isSiteAutomatedTransfer from 'calypso/state/selectors/is-site-automated-transfer';
+import { isJetpackSite } from 'calypso/state/sites/selectors';
+import { getSelectedSiteId } from 'calypso/state/ui/selectors';
+
+type PostByEmailSettingProps = {
+	isFormPending: boolean;
+	address?: string;
+};
+
+const noticeConfig = {
+	duration: 4000,
+};
+
+const moduleSlug = 'post-by-email';
+
+export const PostByEmailSetting = ( { isFormPending, address }: PostByEmailSettingProps ) => {
+	const selectedSiteId = useSelector( getSelectedSiteId ) || 0;
+	const siteIsJetpack = useSelector( ( state ) => isJetpackSite( state, selectedSiteId ) );
+	const siteIsAtomic = useSelector( ( state ) => isSiteAutomatedTransfer( state, selectedSiteId ) );
+	const jetpackRegeneratingPostByEmail = useSelector( ( state ) =>
+		isRegeneratingJetpackPostByEmail( state, selectedSiteId )
+	);
```

Comment:
This is outside the scope of this PR, but in my testing, this selector doesn't seem to work properly (it never returns `true`). The same behavior can be observed in production.

Reply:
Do you think it's worth removing from Post by Email for now, so it's not misleading/confusing for devs reading this code in the future? Or leave it to quietly do nothing in case/until it's fixed in the future?

Reply:
Scratch that question... we were really [only trying to use it for one thing that I decided to remove anyway](https://github.com/Automattic/wp-calypso/pull/90819/files#r1617782153), so that selector is no longer called at all. If it's ever fixed we can bring it back to have parity with Simple sites, but for now we can leave it out, I think.